content/2.0.html.haml

```diff
@@ -90,109 +89,115 @@ title: "Jenkins 2.0"
-      your specific needs. The list of plugins is hand-picked by Jenkins experts
-      from our popular plugins to ensure the best getting-started experience for
-      new users.
-
-    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
-      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
+%h2
+  Pipelines
+
+%h3 Problem
+
+%p As organizations of all types seek to deliver high quality software faster, their use of Jenkins is extending beyond just CI to Continuous Delivery (CD) In order to implement CD, teams  need a flexible way to model and orchestrate their entire delivery pipeline.
+  
+%h3 Solution
+
+%p Jenkins 2.0 now natively supports delivery pipelines as a first-class entity. Pipelines introduces a domain specific language that help Jenkins users to model their software delivery pipelines as code, which can be checked in and version controlled along with the rest of your project's source code.
+
+%img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
+
+%p An expressive domain specific language (configuration as code) makes it easier and faster to create a simple "build, test and deploy" pipeline or build advanced pipelines. The net result is improved productivity for dev and ops teams.
+
+%h4 Key Features and Benefits for Pipelines
+
+%table{:border => 1} 
+  %tr
+    %th Features
+    %th Benefits
+
+  %tr 
+    %td Pipeline as code (DSL)
+    %td
+      %ul
+        %li Easily define simple and complex pipelines through the DSL in a <i> Jenkinsfile</i>.
+        %li Pipeline as code provides a common language to help teams (e.g. Dev and Ops) work together.
+        %li Easily share pipelines between teams by storing common functions in global libraries
+        
+  %tr
+    %td New Pipeline Stage View
+    %td 
+      %ul
+        %li A new interface addition to make it easy to visualize the progress across an entire pipeline.
+        %a{:href => '/images/pipeline-stage-view.png', :target => '_blank'}
+          %img{:src => '/images/pipeline-stage-view.thumb.png', :title => 'A pipeline stage view'}
+ 
+  %tr
+    %td Durable
+    %td
+      %ul
+        %li Pipelines are long lasting and can survive infrastructure outages.
+        
+  %tr      
+    %td Native support for Git, GitHub branches
+    %td 
+      %ul
+        %li Each feature branch can have its own delivery pipeline. 
+  
+%h2 Improved Usability
+
+%h3 Problem
+
+%p The rich plugin ecosystem means that administrators filter through the plugin choices and select those for their users. Once the plugins are installed users have to navigate and complete the multiple plugin configurations choices. This makes the "Getting Started Experience" more involved than it should be.
+
+  
+%h3 Solution
+
+%p For new users, Jenkins 2.0 starts off with set of recommended plugins to help get you started with the right set of tools to get up and running with Jenkins quickly.
```

Comment:
Seems like a lot of repetition here.

For new users, Jenkins 2.0 starts off with set of recommended plugins to help get you started with the right set of tools to get up and running with Jenkins quickly.
->
Jenkins 2.0 guides new users to install a set of recommended plugins, with the option to select further popular plugins used by the Jenkins community, helping users to get up and running quickly.